Foundation/NSString.swift

```diff
@@ -1232,6 +1232,13 @@ extension NSString {
         NSUnimplemented()    
     }
     
+    public convenience init(format: NSString, _ args: CVarArgType...) {
```

Comment:
it is very strange that the importer grabs this as NSString instead of String. But it is in parity with Darwin.

There is also the method
```swift
public convenience init(format: NSString, locale: NSLocale?, _ args: CVarArgType...)
```
as well